src/examples/server/standaloneSseWithGetStreamableHttp.ts

```diff
@@ -0,0 +1,136 @@
+import express, { Request, Response } from 'express';
+import { randomUUID } from 'node:crypto';
+import { McpServer } from '../../server/mcp.js';
+import { StreamableHTTPServerTransport } from '../../server/streamableHttp.js';
+import { ReadResourceResult } from '../../types.js';
+
+// Create an MCP server with implementation details
+const server = new McpServer({
+  name: 'resource-list-changed-notification-server',
+  version: '1.0.0',
+}, {
+  capabilities: {
+    resources: {
+      listChanged: true, // Support notifications for resource list changes
+    },
+  }
+});
```

Comment:
The changes in https://github.com/modelcontextprotocol/typescript-sdk/pull/247 should mean this capability is registered automatically.